test/resources/referral.test.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable func-names,jest/no-disabled-tests */
+import { expect } from 'chai';
+import * as setupPolly from '../setup_polly';
+import EasyPost from '../../src/beta/easypost';
+import Fixture from '../helpers/fixture';
+
+describe('Referral Resource', function () {
+  setupPolly.startPolly();
+
+  before(function () {
+    this.easypost = new EasyPost(process.env.EASYPOST_PROD_API_KEY);
+  });
+
+  beforeEach(function () {
+    const { server } = this.polly;
+    setupPolly.stripCreds(server);
+  });
+
+  async function createTestReferral() {
+    // Note: this returns API keys in plaintext, so do not record a test using this function
+    const referral = await new this.easypost.Referral({
+      name: 'Test Referral',
+      email: 'me@email.com',
+      phone_number: '1111111111', // phone_number will be copied to phone behind the scenes
```

Comment:
I think the API endpoint is returning `phone_number` instead of `phone` now, you can remove the comment.